esmvalcore/preprocessor/_mask.py

```diff
@@ -81,6 +84,14 @@ def mask_landsea(cube, mask_out, always_use_ne_mask=False):
         always apply Natural Earths mask, regardless if fx files are available
         or not.
 
+        .. warning::
+            This option has been deprecated in ESMValCore version 2.5. To
+            always use Natural Earth masks, either explicitly remove all
+            ``ancillary_variables`` from the input cube (when this function is
+            used directly) or specify ``fx_variables: {}`` as option for this
+            preprocessor in the recipe (when this function is used as part of
+            ESMValTool).
```

Comment:
wait - how was it before? if the user didn't specify any `fx_variables` the code was going straight to NE masks no? This behaviour shouldn't change

Reply:
See my table at the bottom: before not using `fx_variables` and using `fx_variables: {}` lead to the same thing (i.e., loading `sftlf` and `sftof`), now `fx_variables: {}` leads to not using any fx file, i.e., going straight to NE.

Reply:
aha, cheers, good table!